RadarSDK/RadarLocationManager.m

```diff
@@ -534,7 +534,7 @@ - (void)handleLocation:(CLLocation *)location source:(RadarLocationSource)source
             }
         }
     } else {
-        stopped = (force || source == RadarLocationSourceVisitArrival);
+        stopped = force;
```

Comment:
@tjulien We need to keep this case. This `else` is used by the `.efficient` preset which on iOS basically just says "rely completely on visits."

Reply:
fixed

Reply:
can we add a comment here about that case?